wsd/Storage.hpp

```diff
@@ -526,20 +532,22 @@ class LocalStorage : public StorageBase
     /// obtained using getFileInfo method
     std::unique_ptr<LocalFileInfo> getLocalFileInfo();
 
-    LockUpdateResult updateLockState(const Authorization&, LockContext&, StorageBase::LockState,
+    LockUpdateResult updateLockState(const Authorization&, LockContext&,
+                                     StorageBase::LockState requestedLockState,
                                      const Attributes&) override
     {
-        return LockUpdateResult(LockUpdateResult::Status::OK);
+        return LockUpdateResult(LockUpdateResult::Status::OK, requestedLockState);
     }
 
-    void updateLockStateAsync(const Authorization&, LockContext&, LockState, const Attributes&,
-                              SocketPoll&,
+    void updateLockStateAsync(const Authorization&, LockContext&, LockState requestedLockState,
+                              const Attributes&, SocketPoll&,
                               const AsyncLockStateCallback& asyncLockStateCallback) override
     {
         if (asyncLockStateCallback)
         {
-            asyncLockStateCallback(AsyncLockUpdate(AsyncLockUpdate::State::Complete,
-                                                   LockUpdateResult(LockUpdateResult::Status::OK)));
+            asyncLockStateCallback(AsyncLockUpdate(
+                AsyncLockUpdate::State::Complete,
+                LockUpdateResult(LockUpdateResult::Status::OK, requestedLockState)));
```

Comment:
Here the given LockState is being used, great. See remark to wsd/Storage.hpp earlier.

Reply:
See reply to related comment.